src/main/java/org/isf/menu/model/User.java

```diff
@@ -45,33 +45,36 @@
 public class User extends Auditable<String> {
 
 	@Id
-	@Column(name="US_ID_A")		
+	@Column(name = "US_ID_A")
 	private String userName;
 
 	@NotNull
 	@ManyToOne
-	@JoinColumn(name="US_UG_ID_A")
+	@JoinColumn(name = "US_UG_ID_A")
 	private UserGroup userGroupName;
 
 	@NotNull
-	@Column(name="US_PASSWD")
+	@Column(name = "US_PASSWD")
 	private String passwd;
 
-	@Column(name="US_DESC")
+	@Column(name = "US_DESC")
 	private String desc;
 
-	@Column(name="US_FAILED_ATTEMPTS")
+	@Column(name = "US_FAILED_ATTEMPTS")
 	private int failedAttempts;
 
-	@Column(name="US_ACCOUNT_LOCKED")
+	@Column(name = "US_ACCOUNT_LOCKED")
 	private boolean isAccountLocked;
 
-	@Column(name="US_LOCK_TIME")
+	@Column(name = "US_LOCK_TIME")
 	private LocalDateTime lockedTime;
 
-	@Column(name="US_LAST_LOGIN")
+	@Column(name = "US_LAST_LOGIN")
 	private LocalDateTime lastLogin;
 
+	@Column(name = "US_DELETED")
+	private boolean deleted = false;
```

Comment:
```suggestion
	private boolean deleted;
```
assigning the value `false` is redundant.